src/store/reducers/todoReducer.ts

```diff
@@ -1,4 +1,12 @@
+import data from '../../api/dummy_todo.json';
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+const initialActiveTab = data.tabData[0].label.toString().toLowerCase();
+
+const dataPropsArray: DataProps[] = data.apiData.map((item) => ({
```

Comment:
import data from '../../api/dummy_todo.json';

dataPropsArray bu function olmamalı. Eğer api data gelirse bu reducer get api data eklemek olmaması lazım.

Reducer içinde data eklenmemeli. todoFetch bu varsa sen sayfadaki gibi todo şu function olarak dispatch get data yapması gerekiyor

Reply:
bunun için biraz bakınıyorum axios kullanmam gerekiyor mu ?

Reply:
bunun için bakıyorum axios kullanmam gerekiyor mu ?

Reply:
Hayır axios gerek yok, ben görevi vermedim...

reducer fake data koyamazsın. senin todo dosyası fake data koyabilirsin demek istedim.